proto/greenfield/storage/events.proto

```diff
@@ -178,6 +178,8 @@ message EventSealObject {
   uint32 global_virtual_group_id = 7;
   // local_virtual_group_id defines the unique id of lvg which the object stored
   uint32 local_virtual_group_id = 8;
+  // for_update indicates whether sealing on a updating object
+  bool for_update = 9;
```

Comment:
I think bool is not good to extend, is it possible it is an Enum here, zero means upload object, 1 means update; we may extend to 2,3,... in the future.

Reply:
will be no longer needed for `EventSealObject` as  `EventUpdateObejctContentSuccess` is also emitted.